src/clj/puppetlabs/services/request_handler/request_handler_core.clj

```diff
@@ -259,21 +248,35 @@
     (= (:type x)
        :puppetlabs.services.request-handler.request-handler-core/bad-request)))
 
+(defn jruby-timeout?
+  [x]
+  "Determine if the supplied slingshot message is for a JRuby borrow timeout."
```

Comment:
Whoops, docstring in wrong place.  Probably not worth the rebase+wait though :sheep: